src/Kiota.Builder/Extensions/StringExtensions.cs

```diff
@@ -266,6 +266,11 @@ private static string NormalizeSymbolsBeforeCleanup(string original)
             result = result.Replace("+", "_plus_", StringComparison.OrdinalIgnoreCase);
         }
 
+        if (result.Contains('\\', StringComparison.OrdinalIgnoreCase))
+        {
+            result = result.Replace(@"\", "Slash", StringComparison.OrdinalIgnoreCase);
```

Comment:
from reading the unit tests this effectively inserts an additional suffix in the request builder's name.
this solution looks disruptive to me, but I'll follow up in the main comment.